x-pack/test/security_solution_endpoint_api_int/apis/endpoint_response_actions/agent_type_support.ts

```diff
@@ -1,34 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { ISOLATE_HOST_ROUTE_V2 } from '@kbn/security-solution-plugin/common/endpoint/constants';
-import { FtrProviderContext } from '../../ftr_provider_context';
-import { targetTags } from '../../../security_solution_endpoint/target_tags';
-
-export default function ({ getService }: FtrProviderContext) {
-  const supertest = getService('supertest');
-
-  describe('Response Actions support for sentinelOne agentType', function () {
-    targetTags(this, ['@ess', '@serverless']);
-
-    describe('and the "responseActionsSentinelOneV1Enabled" feature flag is disabled', () => {
```

Comment:
Why remove the test?  The feature flag is still in our code, just defaulted to `true` now, so until we complete remove that feature flag, I'm thinking that this test should remain, but validate that it does not return an error - right?

Reply:
👍  you're right, I didn't think about reverting the logic. WIll do! thanks!